opennms-webapp-rest/src/test/java/org/opennms/web/rest/support/menu/MenuProviderTest.java

```diff
@@ -29,14 +29,15 @@
 
 import com.google.common.base.Strings;
 import org.junit.Assert;
+
+import static org.hamcrest.Matchers.*;
```

Comment:
nit: It is a test but still want to avoid wildcard imports.

Reply:
Fixed - I need to check my IntelliJ settings :)